tests/integration/helpers/cluster.py

```diff
@@ -172,6 +178,7 @@ def add_instance(self, name, config_dir=None, main_configs=None, user_configs=No
 
         name - the name of the instance directory and the value of the 'instance' macro in ClickHouse.
         config_dir - a directory with config files which content will be copied to /etc/clickhouse-server/ directory
+        base_config_dir - a directory with config.xml and users.xml files which will be copied to /etc/clickhouse-server/ directory
```

Comment:
It's unclear what's the difference between `config_dir` and `base_config_dir`.

Reply:
Actually config_dir is subject to remove. I'll update this text.
Previous version allowed to use config dir from  integration test instead of default one(the one in programs/server). Thus people left it unchanged forever when default config continued to develop.